packages/application-generic/package.json

```diff
@@ -68,7 +68,6 @@
     "@novu/mattermost": "^0.16.0",
     "@novu/ms-teams": "^0.16.0",
     "@novu/netcore": "^0.16.0",
-    "@novu/node": "^0.16.0",
```

Comment:
Application  generic shouldn't have this dependency at all (plus not being used).